product_code/lagom/hl_course_service/impl/src/main/resources/prod-application.conf

```diff
@@ -0,0 +1,29 @@
+include "application"
+
+play {
+  server {
+    pidfile.path = /dev/null
+  }
+
+  http.secret.key = "${APPLICATION_SECRET}"
+}
+
+akka {
+  discovery.method = akka-dns
+
+  management.cluster.bootstrap {
+    contact-point-discovery {
+      discovery-method = kubernetes-api
+      service-name = "hl_course"
```

Comment:
The service is called hlcourse  
Change it here or in the HlCourseService.scala